protocol-specs/generator_outputs/multisignature_registration_transaction/multisignature_transaction.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
you probably want to remove this file now since its a duplicate of  multisignature_registration_transaction.json?

Reply:
Done.